gammapy/spectrum/models.py

```diff
@@ -708,3 +740,33 @@ def plot(self, energy_range, ax=None, energy_unit='TeV',
         if self.scale_logy:
             ax.set_yscale("log", nonposy='clip')
         return ax
+
+
+class AbsorbedSpectralModel(SpectralModel):
+
+    def __init__(self, spectral_model, table_model):
+        """Absorbed spectral model
+
+        Parameters
+        ---------
```

Comment:
One missing `-` character here is making travis-ci fail:
https://travis-ci.org/gammapy/gammapy/jobs/193042173#L1674

Please fix. :-)